sonar-php-plugin/src/main/java/org/sonar/plugins/php/PHPSensor.java

```diff
@@ -80,7 +80,7 @@ public PHPSensor(FileLinesContextFactory fileLinesContextFactory,
   }
 
   public PHPSensor(FileLinesContextFactory fileLinesContextFactory,
-    CheckFactory checkFactory, NoSonarFilter noSonarFilter, @Nullable PHPCustomRulesDefinition[] customRulesDefinitions) {
+    CheckFactory checkFactory, NoSonarFilter noSonarFilter, @Nullable PHPCustomRuleRepository[] customRulesDefinitions) {
```

Comment:
@pynicolas at least in this context I think we can easily rename `customRulesDefinitions` into `customRuleRepositories`

Reply:
Done in 832a63fd.